src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1332,15 +1346,25 @@ describe('ReactDOMComponent', function() {
       ReactDOMServer.renderToString(<input type="text" onclick="1"/>);
       expect(console.error.argsForCall.length).toBe(2);
       expect(
-        console.error.argsForCall[0][0].replace(/\(.+?:\d+\)/g, '(**:*)')
-      ).toBe(
-        'Warning: Unknown DOM property class. Did you mean className? (**:*)'
+        console.error.argsForCall[0][0]
+      ).toContain(
```

Comment:
Can you please keep these all as toBe and check the full error message so we can be confident the message looks right and that it doesn't change accidentally? There are other examples in tests where we do this with stacks.